colibre/config.yml

```diff
@@ -148,11 +148,21 @@ scripts:
     output_file: gas_smoothing_lengths.png
     section: Histograms
     title: Gas Particle Smoothing Lengths
+  - filename: scripts/number_of_agn_thermal_injections.py
+    caption: A histogram showing the cumulative number of black holes with a given total number of thermal energy injections (less than or equal to the number of particles heated) the black hole has had throughout the simulation.
```

Comment:
Can we clarify here that the 'cumulative' is done right-to-left rather than the usual left-to-right?

Reply:
Yes. Done